deployment/README.md

```diff
@@ -1,8 +1,89 @@
-# Build scripts
+# Jellyfin Packaging
 
-All `build.sh` and `package.sh` scripts are for *nix platforms (or WSL on Windows 10).
+This directory contains the packaging configuration of Jellyfin for multiple platforms. The specification is below; all package platforms must follow the specification to be compatable with the central `build` script.
 
-After running both, check the `*/pkg-dist/` folders for the archives and packages.
+## Package List
+
+### Operating System Packages
+
+* `debian-package-x64`: Package for Debian and Ubuntu amd64 systems.
+* `fedora-package-x64`: Package for Fedora, CentOS, and Red Hat Enterprise Linux amd64 systems.
+
+### Portable Builds (archives)
+
+* `debian-x64`: Portable binary archive for Debian amd64 systems.
+* `ubuntu-x64`: Portable binary archive for Ubuntu amd64 systems.
+* `linux-x64`: Portable binary archive for generic Linux amd64 systems.
+* `osx-x64`: Portable binary archive for MacOS amd64 systems.
+* `win-x64`: Portable binary archive for Windows amd64 systems.
+* `win-x86`: Portable binary archive for Windows i386 systems.
+* `win-generic`: Portable binary archive for generic Windows systems.
```

Comment:
what's the difference between `win-generic` and `win-x86`?

Reply:
win-generic, now has the powershell build scripts, and can not run on linux without installing powershell. (And even then)

This will end up containing the script to build the setup, possibly, or we dissolve the powershell scripts into the linux host build and package scripts. We can always ssh into a windows vm (yes win10 has ssh server and client) to do this.

Reply:
Maybe this is better named like `win-setup`?

Reply:
That is not what is does right now though. When we have all the builds setup properly, we won't need the windows packaging scripts.

Reply:
Yea I wasn't sure what this one did, thanka for confirming. For now I can exclude iit from this script. I'll probably add per-platform dependency handling too, so the wcript can know what it needs for each one before starting.

Reply:
I added another section for the "other" builds including Docker, unRaid, Platform, and this one.

Reply:
OK that makes sense, we can get that set up once we figure all that out.